CMakeLists.txt

```diff
@@ -24,6 +24,7 @@ option(ENABLE_HTTP "Enable the use of CURL" ON)
 option(ENABLE_PYTHON_BINDINGS "Enable Python bindings" ON)
 option(ENABLE_NODE_BINDINGS "Build NodeJs bindings" ON)
 option(ENABLE_CCACHE "Speed up incremental rebuilds via ccache" ON)
+option(ENABLE_SCCACHE "Speed up incremental rebuilds via sccache" OFF)
```

Comment:
Maybe mention that this option supersedes `ENABLE_CCACHE` (if that's intentional)